ssa.go

```diff
@@ -175,7 +181,7 @@ func ReadFromSSA(i io.Reader) (o *Subtitles, err error) {
 				format = make(map[int]string)
 				continue
 			default:
-				log.Printf("astisub: unknown section: %s", line)
+				opts.OnUnknownSectionName(line)
```

Comment:
Once `OnUnknownSectionName` is a field, you'll need to wrap this with `if opts.OnUnknownSectionName != nil`